src/dftd4/param.f90

```diff
@@ -67,7 +67,8 @@ subroutine get_rational_damping(functional, param, s9)
    integer :: is, id
    logical :: mbd
 
-   mbd = merge(s9 /= 0.0_wp, .true., present(s9))
+   mbd = .false.
```

Comment:
This changes the default?

Reply:
Why does that matter? The following check for `s9` will always be evaluated (to true if s9 is set).

Reply:
This changes which method is used in case s9 is not provided.

Reply:
What about this? Should be the translation of the merge-statement.
```fortran
if (s9 /= 0.0_wp) then
  mbd = .true.
else
  mbd = present(s9)
endif
```

Reply:
That does still change the behavior. See [merge](https://fortran-lang.org/en/learn/intrinsics/array/#merge).

Why not just use
```f90
mbd = .true.
if (present(s9)) mbd = s9 /= 0.0_wp
```

Reply:
That works. (I thought `merge` takes the mask as first argument like np.where)